sway/tree/view.c

```diff
@@ -1060,6 +1075,14 @@ void view_update_title(struct sway_view *view, bool force) {
 	if (view->foreign_toplevel && title) {
 		wlr_foreign_toplevel_handle_v1_set_title(view->foreign_toplevel, title);
 	}
+
+	if (view->ext_foreign_toplevel) {
+		struct wlr_ext_foreign_toplevel_handle_v1_state foreign_toplevel_state = {
+			.app_id = (char *)view_get_app_id(view),
+			.title = (char *)title,
+		};
+		wlr_ext_foreign_toplevel_handle_v1_update_state(view->ext_foreign_toplevel, &foreign_toplevel_state);
```

Comment:
Suggestion: maybe we can extract this into a helper function?

Reply:
Done